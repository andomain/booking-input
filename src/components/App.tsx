import React from 'react';

import SearchForm from './SearchForm';

import '../styles/styles.scss';

const App = () => (
  <div className="App">
    <SearchForm />
  </div>
);

export default App;
