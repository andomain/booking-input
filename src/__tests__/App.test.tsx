import React from 'react';
import { shallow } from 'enzyme';

import App from '../components/App';

describe('<App />', () => {
  it('Renders a basic application', () => {
    const app = shallow(<App />);
    expect(app.getElements()).toMatchSnapshot();
  })
})
