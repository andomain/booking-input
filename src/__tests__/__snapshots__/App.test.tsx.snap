// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Renders a basic application 1`] = `
Array [
  <div>
    Hello World
  </div>,
]
`;
